pkg/infradriver/ssh_infradriver.go

```diff
@@ -141,11 +142,21 @@ func (d *SSHInfraDriver) GetHostIPListByRole(role string) []net.IP {
 }
 
 func (d *SSHInfraDriver) GetHostEnv(host net.IP) map[string]interface{} {
-	return d.hostEnvMap[host.String()]
+	hostEnv := d.hostEnvMap[host.String()]
+	if _, ok := hostEnv["HostIP"]; !ok {
+		hostEnv["HostIP"] = host.String()
+	}
+	return hostEnv
 }
 
 func (d *SSHInfraDriver) GetClusterEnv() map[string]interface{} {
-	return d.clusterRenderData
+	if _, ok := d.clusterEnv["RegistryDomain"]; !ok {
+		d.clusterEnv["RegistryDomain"] = "sea.hub"
```

Comment:
Strings like "sea.hub", "5000", should be consts and put into well known common file.